server/fleet/orbit.go

```diff
@@ -0,0 +1,17 @@
+package fleet
+
+import "encoding/json"
+
+// OrbitConfigNotifications are notifications that the fleet server sends to
+// fleetd (orbit) so that it can run commands or more generally react to this
+// information.
+type OrbitConfigNotifications struct {
+	RenewEnrollmentProfile bool `json:"renew_enrollment_profile,omitempty"`
+}
+
+type OrbitConfig struct {
+	Flags         json.RawMessage          `json:"command_line_startup_flags,omitempty"`
+	Extensions    json.RawMessage          `json:"extensions,omitempty"`
+	NudgeConfig   json.RawMessage          `json:"nudge_config,omitempty"`
```

Comment:
Any reason to declare it as `json.RawMessage`? `Flags` and `Extensions` are not under our control, I believe (at least for flags, as those as the osquery flags and we don't want to block any setting that could be supported by osquery that we don't know about), but that's not the case for nudge config?

Reply:
Nevermind, just saw later on that this is because what's provided here is the result of executing the nudge config template with our specifically-supported fields!